profiles/schema.py

```diff
@@ -0,0 +1,76 @@
+import graphene
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
+    class Meta:
+        model = DivisionOfInterest
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+
+class ProfileInput(graphene.InputObjectType):
+    user_id = graphene.ID()
+    nickname = graphene.String(required=True)
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = graphene.String()
+    contact_method = graphene.String()
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    ok = graphene.Boolean()
```

Comment:
now with more experience with graphene, I see that this `ok` field is not obligatory in any waym and was just in the docs for the purposes of a more detailed example, we can drop it for now. since, there is no logic where the client would check if ok is True and we don't even have logic for `ok=False`.

Reply:
This is the 6th time I am unresolving this 😅 😅 😅

Reply:
I meant, that we should get rid of this `ok` stuff.

Reply:
> we can drop it for now

Reply:
Didnt notice you reopened it, i just thought I hadn't closed it...